src/uv-common.c

```diff
@@ -67,6 +67,14 @@ static uv__allocator_t uv__allocator = {
   free,
 };
 
+char* uv__strdup(const char* s) {
```

Comment:
Active code does not use ``uv__strdup`` except ``unix/darwin.c``.
I would like to not add to here but add to unix/darwin.c as a local function.
I want to minimize as best as we can at real binary size.